packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -674,22 +674,32 @@ export class RunningSummarizer implements IDisposable {
 						});
 						await delay(delaySeconds * 1000);
 					}
-				}
 
-				if (this.shouldAbortOnSummaryFailure) {
-					this.mc.logger.sendTelemetryEvent(
-						{
-							eventName: "ClosingSummarizerOnSummaryStale",
-							reason,
-							message: lastResult?.message,
-						},
-						lastResult?.error,
-					);
+					if (this.shouldAbortOnSummaryFailure) {
+						// There's no need to delay here as we already have summarizeAttemptDelay given to us by the server.
+						if (delaySeconds === 0) {
+							const closeSummarizerDelayOverrideMs =
+								this.mc.config.getNumber(
+									"Fluid.ContainerRuntime.Test.CloseSummarizerOnSummaryStaleDelayOverrideMs",
+								) ?? defaultCloseSummarizerDelayMs;
+							await delay(closeSummarizerDelayOverrideMs);
+						}
+
+						this.mc.logger.sendTelemetryEvent(
+							{
+								eventName: "ClosingSummarizerOnSummaryStale",
```

Comment:
I think terms "ClosingSummarizerOnSummaryStale" & "latestSummaryStateStale" do not reflect reality. Nothing is really stale. Summary failed, and without cracking error messages we do not know why, so I feel like code should not come up with guesses. It could be 400/404/429 from service.

Reply:
At this point, we have already sent a telemetry event `Summarize_cancel` which contains all that information. We do not need to send multiple error messages that we do not read that propagate that information. At this point, we just want to know if the feature is engaged, and thus the telemetry event.